fetch_test.go

```diff
@@ -56,6 +58,26 @@ func TestClone_unpacked(t *testing.T) {
 	rmDir(t, cloneAndCheckout(t, "ipfs://ipfs/QmYFpZJs82hLTyEpwkzVpaXGUabVVwiT8yrd6TK81XnoGB/unpackedTest", expectedClone))
 }
 
+func TestClone_ipnsPublished(t *testing.T) {
+	bareHash := "Qmf4ZzbmnqyKEjuBH3hNpQWu1fMUZWdy91DwAsUXXc4Kw1" // == /ipfs/QmYFpZJs82hLTyEpwkzVpaXGUabVVwiT8yrd6TK81XnoGB/unpackedTest
+	id, err := ipfsShell.ID()
+	checkFatal(t, err)
+	//err = ipfsShell.Publish(id.ID, bareHash)
+	err = ipfsShell.Publish("", bareHash)
```

Comment:
@whyrusleeping I can't get this to work. 

L65 blocks indefinitely and L66 gives me `failed to find any peer in table`.

Any Idea how I could test this without relying on a remote node?